integration_tests/protractor.conf.js

```diff
@@ -0,0 +1,32 @@
+const ConsoleReporter = require('jasmine-console-reporter');
+
+exports.config = {
+    framework: 'jasmine',
+    suites: {
+        base: ['tests/base.scenario.js']
+    },
+    capabilities: {
+        browserName: 'chrome',
+        chromeOptions: {
+            args: [
+              '--disable-gpu',
+              '--no-sandbox',
+              '--window-size=1920,1200',
+              '--disable-background-timer-throttling',
+              '--disable-renderer-backgrounding',
+              '--disable-raf-throttling',
+            ],
+            prefs: {
+              'profile.password_manager_enabled': false,
+              'credentials_enable_service': false,
+              'password_manager_enabled': false,
+            },
+        },
+    },
+    onPrepare: () => {
+        browser.waitForAngularEnabled(false);
```

Comment:
Most of the tutorials mention using `browser.ignoreSynchronization = true;` instead of this.